docs/overview/cvct_mixing_topo.rst

```diff
@@ -0,0 +1,25 @@
+Convection and mixing over topography
+-------------------------------------
+
+
+Dense plumes generated by localized cooling on the continental shelf of the
+ocean may be influenced by rotation when the deformation radius is smaller
+than the width of the cooling region. Rather than gravity plumes, the
+mechanism for moving dense fluid down the shelf is then through geostrophic
+eddies. The simulation shown in :numref:`fig_convect_and_topo`
+(blue is cold dense fluid, red is
+warmer, lighter fluid) employs the non-hydrostatic capability of MITgcm to
+trigger convection by surface cooling. The cold, dense water falls down the
+slope but is deflected along the slope by rotation. It is found that
+entrainment in the vertical plane is reduced when rotational control is
+strong, and replaced by lateral entrainment due to the baroclinic
+instability of the along-slope current.
+
+  .. figure:: figs/plume.*
+    :width: 70%
+    :align: center
+    :alt: Finit volume techniques
```

Comment:
typo: "finit volume"

This one is my fault, sorry. But, is this the best alt text to have for this figure? Perhaps "Non-hydrostatic plume over a shelf" would be better.